### PR TITLE
OSPP 2021: mid-term report of wechaty-cli

### PR DESCRIPTION
**Pull request by chinggg**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Review by lijiarui — APPROVED** [review]
LGTM!

Please get your mentor's approvement then I will merge this PR.

**Review by huan — APPROVED** [review]
LGTM

Thank you very much for sharing the CLI project for the community!

File: jekyll/_posts/2021-08-26-ospp-mid-term-cli.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+项目仓库: <https://github.com/wechaty/cli>  
+
+### live coding视频
+
+{% include iframe.html src="https://www.youtube.com/watch?v=1U0ONeHV7z8" %}
+
+### PPT展示视频
+
+{% include iframe.html src="https://www.youtube.com/watch?v=xPsAwLglVdM" %}
+
```
Comment (lijiarui) [inline jekyll/_posts/2021-08-26-ospp-mid-term-cli.md:87]: Please add Chinese version video as https://github.com/wechaty/summer/issues/80#issuecomment-914480564 request, and add it to the blog: 


```
> 国内链接：https://www.bilibili.com/XXXXXXXXXXXXXXXXXXXXXXX
```

File: jekyll/_posts/2021-08-26-ospp-mid-term-cli.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  - 考虑到 blessed 和终端的特性，一般聊天软件的设计（如新对话置顶）可能无法照搬
+  - 在保证可用性的情况下，使用 React 和 Redux 重构项目
+
+## 项目成果
+
+项目仓库: <https://github.com/wechaty/cli>  
+
+### live coding视频
+
+{% include iframe.html src="https://www.youtube.com/watch?v=1U0ONeHV7z8" %}
```
Comment (lijiarui) [inline jekyll/_posts/2021-08-26-ospp-mid-term-cli.md:80]: Please add Chinese version video as https://github.com/wechaty/summer/issues/80#issuecomment-914480564 request, and add it to the blog: 


```
> 国内链接：https://www.bilibili.com/XXXXXXXXXXXXXXXXXXXXXXX
```

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2021-08-26-ospp-mid-term-cli.md (hunk 2 above)
Comment (chinggg) [reply jekyll/_posts/2021-08-26-ospp-mid-term-cli.md:80]: Will add Chinese version video tomorrow

**Comment by lijiarui** [comment]
@chinggg Please make GitHub action green.

**Comment by chinggg** [comment]
> Please add Chinese version video as wechaty/summer#80 (comment) request, and add it to the blog:

> 国内链接：https://www.bilibili.com/XXXXXXXXXXXXXXXXXXXXXXX

@lijiarui The style of bare url is not allowed by the CI rule.

**Comment by lijiarui** [comment]
@chinggg You can follow other students did: https://github.com/wechaty/wechaty.js.org/pull/1236/files

**Comment by huan** [comment]
Link to https://github.com/wechaty/summer/issues/80

**Review by lijiarui — APPROVED** [review]
LGTM!

**Pull request merged**